### ISSUE TITLE
`ext_proc_integration_test` is v flakey

### ISSUE THREAD

**Issue by phlax** [envoyproxy/envoy#28568]
```console
//test/extensions/filters/http/ext_proc:ext_proc_integration_test       TIMEOUT in 1 out of 2 in 914.4s
```

https://dev.azure.com/cncf/envoy/_build/results?buildId=143656&view=logs&j=d1f76054-8f79-554b-6f4a-11d6a63b8e00&t=a193292f-96b1-53c3-0505-a923ddcc3f84&l=4790

Its been happening since yesterday

![image](https://github.com/envoyproxy/envoy/assets/454682/c1f62b70-b283-4614-92df-1659e21275a0)

and is almost certainly from:

https://github.com/envoyproxy/envoy/pull/27812

**Comment by phlax** [comment]
cc @yanjunxiang-google @yanavlasov 

i think we want to resolve this asap due to pending release - ie revert or fix forward

**Comment by phlax** [comment]
also seeing it on PRs

**Comment by yanjunxiang-google** [comment]
@phlax I think we can fix forward. This is the  PR: https://github.com/envoyproxy/envoy/pull/28574

**Comment by phlax** [comment]
Fixed in #28574
